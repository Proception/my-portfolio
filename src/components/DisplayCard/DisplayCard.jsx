import React, { Component } from 'react';

const DisplayCard = (props) => {
  return (
    <div class= {props.class}>
      {props.children}
    </div>
  );
};

export default DisplayCard;
