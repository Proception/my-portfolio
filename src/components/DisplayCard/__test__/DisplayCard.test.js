import React from 'react';
import renderer from 'react-test-renderer';
import DisplayCard from '../DisplayCard.jsx';


test('DisplayCard component', () => {
  const component = renderer.create(
    <DisplayCard />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
