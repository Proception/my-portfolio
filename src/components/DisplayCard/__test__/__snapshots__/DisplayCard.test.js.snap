// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayCard component 1`] = `<div />`;
